packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -316,6 +320,7 @@ export class AtomicBreadcrumbManager {
     return (
       <span
         class="pl-1 text-primary align-baseline"
+        aria-label="Clear All Filters"
```

Comment:
Is it necessary to have the `mainClear` wrapped in a span ? 
And if so, can't the aria-label be set on the button itself ?

Just to say that we have to be careful about just "reproducing what JSUI has", because many of the labels and accessibility properties that exists in JSUI are there more as a crutch/workaround, instead of being absolutely necessary.

Reply:
We should either be using a button with the SVG inside it or adding `role=button` I think https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role .

Reply:
First, this `aria-label` should not have been there, it is supposed to be on the `Clear All Filters` button. 

As for the `span` tag, it is necessary to have a have a wrapper for the svg. According to the documentation of the plugin, the svg is to be inserted as the `innerHTML` property ([stencil-inline-svg](https://www.npmjs.com/package/stencil-inline-svg))

I think adding `role="button"` is a good choice.

Reply:
Actually, changing it to `button` instead of `span` works.

Reply:
Having the svg inside a `button` tag makes the navigation less intuitive. For example, if I use "tab" to navigate through the breadcrumbs, it goes to the breadcrumb (e.g. `0-20`), then by pressing "tab" again, it goes to the svg button. I don't think this is ideal.